docs/images/siddhi-logo.svg

```diff
@@ -0,0 +1,19 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Year should be updated

Reply:
Fixed by 71dad9c6b3c676a5a94d62bcb9ce7fcd5b91e918